build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+# Sets the shell to be PowerShell 
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC for ovs
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait; \
+    rm C:\vc_redist.x64.exe
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip; \
+    Expand-Archive C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip -DestinationPath C:\  ; \
+    rm C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip
+
+# Download, extract OpenSSL and copy libraries
+RUN curl.exe -Lo C:\openssl-1.0.2u-x64_86-win64.zip https://indy.fulgan.com/SSL/openssl-1.0.2u-x64_86-win64.zip; \
+    Expand-Archive C:\openssl-1.0.2u-x64_86-win64.zip -DestinationPath C:\  ;  \
+    Move-Item -Path 'C:/libeay32.dll', 'C:/ssleay32.dll' -Destination 'C:/Windows/System32/' ; \
+    mkdir C:/openvswitch/var/run/openvswitch ;\
```

Comment:
I think it's wrong to create this directory in the image's filesystem as this will store OVS conf.db, which is supposed to be persistent on host filesystem. Creating this folder in the image means the conf file will exist in container only and will be lost once container restarts.

Reply:
We had discussed on this folder creation recently as this folder is not used inside of container, conf.db file will use host folder, will remove this folder creation step ,also it is not used,

Reply:
> I think it's wrong to create this directory in the image's filesystem as this will store OVS conf.db, which is supposed to be persistent on host filesystem. Creating this folder in the image means the conf file will exist in container only and will be lost once container restarts.

One correction - this folder is not for conf.db file ,
The var/run/openvswitch directory is used by ovs to store various runtime files and information. It may contain files such as sockets for communication between different components of ovs, process ID (PID) files, and other runtime data.

For example, the ovsdb-server process, which is the database server component of Open vSwitch, may create a Unix domain socket file (e.g., db.sock) inside the var/run/openvswitch directory for communication with other processes.
If this folder is not available,  ovsdb-server does not start, 
if the folder C:/openvswitch/var/run/openvswitch is created by Install OVS script on host, and  ovsdb-server can find this then it can be removed

Reply:
@Atish-iaf , confirmed that C:/openvswitch/var/run/openvswitch is created by install script on host and used by ovsdb-server  , hence no need to have it created inside container.

Reply:
That depends on how you will set conf.db's folder in another PR. On Linux, we use the same folder which stores socket and PID: https://github.com/antrea-io/antrea/blob/f32f7963842617487c5859b0cb28e789af7132f0/build/images/scripts/start_ovs#L22-L23